### PR TITLE
improve jira/zoom docs 

### PR DESCRIPTION
**Pull request by eschultink**
### Features
 - clarify Jira docs; clarify that terminal is required, `curl` doesn't work out of the box on Windows ...
 - screenshots, alignment on Zoom docs

### Change implications

 - dependencies added/changed? **no**
 - something to note in `CHANGELOG.md`? **no**

### CODE REVIEW

File: docs/sources/atlassian/jira/README.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ And following granular scopes:
   - `read:group:jira`: for retrieving group members
   - `read:avatar:jira`: for retrieving group members
 
+You will need a web browser and a terminal with `curl` available (such as macOS terminal,
```
Comment (khnumdev) [inline docs/sources/atlassian/jira/README.md:20]: I'll add a note that `curl` is installed by default in Windows 10/11 -> https://curl.se/windows/microsoft.html.

Tested with my own Windows 11 and is installed in the powershell terminal

**Review by khnumdev — APPROVED** [review]

**Review by aperez-worklytics — APPROVED** [review]

File: docs/sources/atlassian/jira/README.md (hunk 1 above)
Comment (eschultink) [reply docs/sources/atlassian/jira/README.md:20]: ok, interesting ... one Jira admin said that the commands didn't work for him, and had to enlist help from someone with a Mac.

Comment (aperez-worklytics) [reply docs/sources/atlassian/jira/README.md:20]: ok, let met check that later. Maybe PS version is an alias of `invoke-weburl` command, so the parameters may differ about what we have in the docs.

Comment (khnumdev) [reply docs/sources/atlassian/jira/README.md:20]: Yeah, from terminal/PS I had the issue described in the link:

```
cmdlet Invoke-WebRequest at command pipeline position 1
Supply values for the following parameters:
Uri:
```

So I think that adding that link as part of the documentation could help on this.

**Pull request merged**